src/apps/replication/lib/replica_learn.cpp

```diff
@@ -212,7 +214,7 @@ void replica::on_learn(dsn_message_t msg, const learn_request& request)
     auto it = _primary_states.learners.find(request.learner);
     if (it == _primary_states.learners.end())
     {
-        response.err = (response.config.status == PS_SECONDARY ? ERR_OK : ERR_OBJECT_NOT_FOUND);
+        response.err = ERR_OBJECT_NOT_FOUND;
```

Comment:
but by the above _primary_states.get_replica_config(), the response.config.status is always PS_POTENTIAL_SECONDARY...